Formula/alot.rb

```diff
@@ -0,0 +1,102 @@
+class Alot < Formula
+  desc "Terminal-based mail user agent for the notmuch mail system"
+  homepage "https://github.com/pazz/alot"
+  url "https://github.com/pazz/alot/archive/0.3.7.tar.gz"
+  sha256 "fd2d4121ba109224560919e84e320b09526891fbc0038afcea26a3f1284dad26"
+  head "https://github.com/pazz/alot.git", :branch => "testing"
+
+  option "without-docs", "Don't build documentation"
+
+  depends_on "gpgme"
+  depends_on "libmagic"
+  depends_on "notmuch"
+  depends_on "sphinx-doc" => :build if build.with? "docs"
```

Comment:
`depends_on "sphinx-doc" => [:build, :recommended]` and do `option "without-sphinx-doc", "Don't build documentation"` above.